src/software/ai/navigator/path_planner/enlsvg_path_planner_test.cpp

```diff
@@ -818,3 +818,86 @@ TEST_F(TestEnlsvgPathPlanner, test_enlsvg_path_planner_close_start_end_but_block
     TestUtil::checkPathDoesNotIntersectObstacle(
         {path_points.begin() + 1, path_points.end()}, obstacles);
 }
+
+TEST_F(TestEnlsvgPathPlanner, test_enlsvg_path_planner_simulated_hrvo)
+{
+    Field field        = Field::createSSLDivisionBField();
+    Team friendly_team = Team(Duration::fromMilliseconds(1000));
+    Team enemy_team    = Team(Duration::fromMilliseconds(1000));
+    Ball ball          = Ball(Point(), Vector(), Timestamp::fromSeconds(0));
+    World world        = World(field, ball, friendly_team, enemy_team);
```

Comment:
nit: we don't need these variables, all we need is field (replace world.field() with field), same for the tests below

Reply:
done